ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -50,6 +53,8 @@ ci/bazelisk.sh test \
     //sw/device/silicon_creator/lib/drivers:hmac_functest_fpga_cw310 \
     //sw/device/silicon_creator/lib/drivers:retention_sram_functest_fpga_cw310 \
     //sw/device/silicon_creator/lib/drivers:uart_functest_fpga_cw310 \
+    //sw/device/silicon_creator/lib/drivers:watchdog_functest_fpga_cw310 \
+    //sw/device/silicon_creator/mask_rom:e2e_bootup_no_rom_ext_signature_fpga_cw310 \
```

Comment:
Sorry I didn't catch that this was a mask rom test when you asked about adding it.